keyboards/cipulot/ec_tkl/info.json

```diff
@@ -0,0 +1,434 @@
+{
+    "manufacturer": "Cipulot",
+    "keyboard_name": "EC TKL",
+    "maintainer": "Cipulot",
+    "bootloader": "stm32-dfu",
+    "build": {
+        "lto": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096
+        }
+    },
+    "features": {
+        "bootmagic": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "processor": "STM32F411",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 22
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x6BC3",
+        "shared_endpoint": {
+            "keyboard": true
+        },
+        "vid": "0x6369"
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B4"
+    },
+    "community_layouts": ["tkl_jis", "tkl_iso_tsangan", "tkl_ansi_tsangan"],
+    "layouts": {
```

Comment:
does this EC variant support [`tkl_ansi`](/qmk/qmk_firmware/tree/master/layouts/default/tkl_ansi) and [`tkl_iso`](/qmk/qmk_firmware/tree/master/layouts/default/tkl_iso) Community Layouts?

Reply:
It does, yup, missed the addition of that in the list. Actual supported layouts shared in pics replying to another review here

Reply:
I didn't add them in the explicit keymaps since the other options listed are what 90% of the time are used and didn't wanted to add even more layouts than what the average user of the board will use.